src/yunohost/app.py

```diff
@@ -338,13 +334,12 @@ def app_list(filter=None, raw=False, installed=False, with_backup=False):
     return {'apps': list_dict} if not raw else list_dict
 
 
-def app_info(app, show_status=False, raw=False):
```

Comment:
You should remove the `-s` in this file:

https://github.com/YunoHost/yunohost/blob/cb247a8140d9ee8a71c663a2da9b3d106ef969bf/data/actionsmap/yunohost.yml#L591-L594

Reply:
Ah yes nice catch, thanks ! :+1: